File: CSharp/Syntax/IfBLock.cs
using static System.Console;

public class Program {
	public static void Main() {
		for (var iA = 0; iA <= 30; iA++) {
			if (iA % 3 == 0 || iA % 4 == 0) {
				WriteLine(iA + " é múltiplo de 3 ou de 4");
			}
		}
	}
}

//https://pt.stackoverflow.com/q/199299/101

File: JavaScript/Syntax/IfBlock.js
if (false)
    console.log(1);
    console.log(2);

if (false) {
    console.log(1);
    console.log(2);
}

if (false)
    if (true)
        console.log(1);
else
    console.log(2);

if (false) {
    if (true) {
        console.log(1);
    } else {
        console.log(2);
    }
}

if (false)
    if (true)
        console.log(1);
    else
        console.log(2);

let x = 1;
{
   let x = 2;
   console.log(x);
}
console.log(x);

//https://pt.stackoverflow.com/q/450094/101
